### DISCUSSION TITLE
Новый закон господа

### DISCUSSION THREAD

**Discussion by Emopunkkk** [Flowseal/zapret-discord-youtube#990] [General]
РКН начнет следить за пользователями, которые пытаются обойти блокировки — операторов связи обяжут собирать всю инфу о таких случаях и передавать её ведомству.

Что написано в пояснении к новому закону:

Эта мера позволит предотвратить доступ к информации, распространение которой ограничено на территории России, и защитить граждан от возможного вреда.

Если приказ вступит в силу, то ведомству в течение 15 дней будут докладывать о пользователях, которые заходят на заблокированные в России веб-ресурсы.

**Comment by PocoX3Obed** [comment]
А как же интересно операторы связи будут следить за обходами, которые эти слежки и обходят?

**Comment by erirosen** [comment]
Для такого есть раздел дискуссий, а ты сейчас в разделе репорта багов.

**Comment by TakehisaYuji** [comment]
в тюрьмах места не хватит

**Reply by erirosen** [reply]
Да и придётся также следить за Медведевыми, Рогозинами и прочими личностями, кто несмотря на блокировку Твиттера каким-то магическим образом продолжает им пользоваться. Хотя кого я обманываю, власть только простых людей душит.

**Comment by JaMiNkL** [comment]
уже сами ркн опровергли... "Тут три варианта: или испугались реакции и переобулись, или они не намерены, а уже собирают, или просто сейчас крутят задом, как с "блокировкой" Ютуба и это будет не сбор информации, а коллекция данных" ©ИнсайдерКэм

**Comment by OlehanDROchill** [comment]


**Reply by erirosen** [reply]
Доказательства есть?
